src/node.rs

```diff
@@ -101,6 +101,9 @@ impl NodeBuilder {
         let full_id = FullId::new();
 
         StateMachine::new(move |crust_service, timer| {
+            let not_allowed_on_lan = self.deny_other_local_nodes &&
```

Comment:
This change was necessary to avoid `rustfmt` getting all confused about line 118. It was inserting a space at the end of the line after `} else if` and failing 👿